aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -211,6 +237,28 @@ module aptos_framework::aptos_governance {
         borrow_global<GovernanceConfig>(@aptos_framework).required_proposer_stake
     }
 
+    #[view]
+    /// Return remaining voting power of a stake pool on a proposal.
+    public fun get_remaining_voting_power(stake_pool: address, proposal_id: u64): u64 acquires VotingRecords, VotingRecordsV2 {
+        if (features::partial_governance_voting_enabled()) {
```

Comment:
I think it's cleaner to create a function is_partial_voting_enabled() which checks both the feature flag and that VotingRecordsV2 is initialized